app/views/idv/doc_auth/_combined_upload.html.erb

```diff
@@ -0,0 +1,74 @@
+<% title t('titles.doc_auth.upload') %>
+
+<%= render 'idv/doc_auth/error_messages', flow_session: flow_session %>
+
+<%= render PageHeadingComponent.new do %>
+  <%= t('doc_auth.headings.combined_upload') %>
+<% end %>
+
+<p>
+  <%= t('doc_auth.info.combined_upload') %>
+</p>
+
+<div class="grid-row grid-gap grid-gap-1">
+  <div class="grid-col-12 tablet:grid-col-auto">
+    <%= image_tag(
+          asset_url('idv/phone-icon.svg'),
+          alt: t('image_description.camera_mobile_phone'),
+          width: 88,
+          height: 88,
+        ) %>
+  </div>
+  <div class="grid-col-12 tablet:grid-col-fill">
+    <div class="usa-tag usa-tag--informative">
+      <%= t('doc_auth.info.tag') %>
+    </div>
+    <h2 class="margin-y-105">
+      <%= t('doc_auth.headings.combined_upload_from_phone') %>
```

Comment:
In the future, maybe we circle back to unqualify these as "combined", since once the feature is enabled and the feature falg removed, I don't think we need to consider this as anything other than the default text for the screen.

Reply:
Yeah that was the plan. Anything with "combined" will be renamed, including querystring params.